src/renderer/views/AppSettings/AppSettings.tsx

```diff
@@ -1,6 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './AppSettings.scss';
+import { useDispatch, useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import { Button, MenuItem, Select } from '@mui/material';
+import {
+  selectCurrentLocale,
+  setLocale,
+} from '../../../shared/redux/slices/settingsSlice';
 
-const AppSettings = () => <div>AppSettings</div>;
+const AppSettings = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const [language, setLanguage] = useState(useSelector(selectCurrentLocale));
+
+  function submitChanges() {
+    dispatch(setLocale(language));
+  }
+
+  return (
+    <div>
+      {t('AppSettings.title')}
+      <Select
+        label={t('AppSettings.language')}
```

Comment:
For accessibility reasons it might be worth giving this component an `aria-label` or `title` prop. I think the `label` prop in MUI is not for accessibility but rather for some sort of internal layout magic. Rest of the accessibility props are handled by MUI internally (focusing, roles and other aria tags).

Reply:
Did not pay a lot of attention to that as the view is just a placeholder now, but I guess it is worth to fix that, if only to not introduce a bad example.